interfaces/vrep/DQ_VrepRobot.m

```diff
@@ -30,7 +30,16 @@
 % DQ Robotics website: dqrobotics.sourceforge.net
 %
 % Contributors to this file:
-%     Murilo Marques Marinho - murilo@nml.t.u-tokyo.ac.jp
+%     1. Murilo Marques Marinho - murilo@nml.t.u-tokyo.ac.jp
+%       - Responsible for the original implementation.
+%     2. Frederico Fernandes Afonso Silva (frederico.silva@ieee.org)
+%       - Altered the names of the following methods to ensure
```

Comment:
Careful with the backwards compatibility. It is better to add an alias for the method than to rename it.

Reply:
Backwards compatibility added in [a02ad1b](https://github.com/dqrobotics/matlab/pull/108/commits/a02ad1ba01016269077e04b6465c43d803049bd1).